### PR TITLE
chore:refactor var/secrets

### PR DESCRIPTION
**Pull request by sr4850**
## Description

Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)

## Before submitting (or marking as "ready for review")

- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [ ] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: .github/workflows/e2etest.yaml [hunk 1]
```diff
@@ -108,31 +113,31 @@ jobs:
...
+                  "value": "${{ github.event.inputs.cucumber_tags || inputs.cucumber_tags }}"
                 },
                 {
                   "name": "buildId",
                   "value": "${{ env.JOB_NAME }}"
                 },
                 {
                   "name": "mavenOptions",
-                  "value": "${{ inputs.maven_options }}"
+                  "value": "${{ github.event.inputs.maven_options || inputs.maven_options }}"
```
Comment (JoshuaLicense) [inline .github/workflows/e2etest.yaml:141]: Isn't `github.event.inputs` the same as `inputs` in this context?

File: .github/workflows/e2etest.yaml
Comment (JoshuaLicense) [inline .github/workflows/e2etest.yaml:1]: Minor, but there's nothing that warrants being in `secrets` in this workflow. Nothing is sensitive in this file.

**Review by JoshuaLicense — COMMENTED** [review]
Looks good, some questions

File: .github/workflows/e2etest.yaml (hunk 1 above)
Comment (sr4850) [reply .github/workflows/e2etest.yaml:141]: Thought we needed it if we are to invoke directly from the mono-repo, but if we are fine with just inputs can revert it back

Comment (JoshuaLicense) [reply .github/workflows/e2etest.yaml:141]: All good from the mono-repo, we can use `inputs` directly for both events.

**Review by JoshuaLicense — APPROVED** [review]
Looks good, nice work! Thanks for doing the changes!

**Comment by sdh100shaun** [comment]
we are going to want to change the queue and task definition at some point

**Pull request merged**